### ISSUE TITLE
I downloaded the ms-celeb data and trained it as train_softmax.py

### ISSUE THREAD

**Issue by haha23h** [davidsandberg/facenet#326]
**train_softmax.py  argument :**
/app/face_detect/src/train_softmax.py --logs_base_dir=/data/storage/logs/facenet --models_base_dir=/data/storage/fd_model/20170512-110547 --data_dir=/data/storage/facenet/MsCelebV1_crop  --image_size=160 --model_def=models.inception_resnet_v1 --lfw_dir=/data/storage/logs/lfw_new_validation/lfw_mtcnnpy_160/ --optimizer=RMSPROP --learning_rate=-1 --keep_probability=0.8 --random_crop --random_flip --learning_rate_schedule_file=/data/storage/logs/lfw/learning_rate_schedule_classifier_msceleb.txt --weight_decay=2e-4 --center_loss_factor=5e-5 --center_loss_alfa=0.9 --lfw_pairs=/data/storage/logs/lfw_new_validation/pairs.txt --gpu_memory_fraction=0.7 --epoch_size=250 --max_nrof_epochs=1000 --lfw_batch_size=100

**lfw_result :** 
60000	0.94083	0.48933
60250	0.94033	0.41967
60500	0.94033	0.47567
60750	0.94233	0.43267
61000	0.94067	0.48867
61250	0.94217	0.42200
61500	0.94267	0.51633
61750	0.93717	0.48200
62000	0.93950	0.41367
62250	0.94200	0.42067
62500	0.94233	0.49367
62750	0.50000	0.00000
63000	0.50000	0.00000
63250	0.50000	0.00000
63500	0.50000	0.00000
63750	0.50000	0.00000
64000	0.50000	0.00000
64250	0.50000	0.00000
64500	0.50000	0.00000
64750	0.50000	0.00000

From epoch 62500 lfw results in 0.00000.
I have an incorrect result, but I do not know why.
I would like to get results similar to the model-20170512-110547.meta model.

**Comment by davidsandberg** [comment]
Hi @haha23h,
When you change epoch_size you also change how the learning rate schedule behaves. It should be set to 1000. Also, that learning rate schedule has only been defined up to 250 epochs. But most importantly you have to do some cleaning of the MsCeleb dataset. I used calculate_filtering_metrics.py but I haven't had time to write a wiki page about it yet.
